### DISCUSSION TITLE
Disable SOS call

### DISCUSSION THREAD

**Discussion by Tony2k** [cedricp/ddt4all#1184] [Q&A] [answered]
I replaced the original radio of the Dacia Spring with a generic one (non-can) and now the writing "check sos call" appears on the panel.  Why does the call have to go through the radio if it is not equipped with GPS or a telephone module?  Is it possible to disable this function with ddt4all?

**Comment by AndyIreland** [comment]
could be if you can unlock the security gateway (if its got one) with a token, and then do a search in DDT4all and if it finds 'Clustre' look on settings and see if you can turn off the SOS message on the dash

**Comment by AndyIreland** [comment]
in my Renault Zoe EV there is some ecall settings in the rlink1 medianav MFD  settings:  I couldnt say where it is on your dacia spring . Maybe in the easylink radio or whatever you have? 

![ecall](https://github.com/cedricp/ddt4all/assets/49172217/446afbbb-0fbb-4fe3-a47b-5d26247e33ed)

**Comment by Tony2k** [comment, marked as answer]
I solved the problem working on DTC Activation Status:

![Spring](https://github.com/cedricp/ddt4all/assets/16784561/c001290b-3500-446f-ba5e-0e8e9e3e5ece)

**Comment by rxtarcher** [comment]
Hello, I got exactly the same message "Check SOS Call".

I am trying to go through ddt4all in order to deactivate "Aux-In Info/Tel-Mute-Output" and "Virgin Configuration", but it doesn't work. I press the "Einstein" button in order to activate expert mode and change the value from "Activated" to "Deactivated", I press "Write activation status of DTC" and then "Key On/Off Reset" but nothing happens. If I refresh or close and open ddt4all both options are still "Activated". 
Also, in the "Read" right column, I can't see if an option is "Activated" or "Deactivated", they are all write "NO DATA".

Do you know what do I have to do in order to be able to deactivate both options?

Thank you.

**Reply by matzel82** [reply]
Sometimes the integrated battery or ECU

**Comment by AndyIreland** [comment]
I presume car more than likely has a TCU ? (telecommunications ECU) in which case you might have to disable SOS call from those settings too. And turn off SOS through the DDT4ALL Cluster settings - you then might even have to clear DTC errors that are showing up - to make that message go away.

**Reply by rxtarcher** [reply]
Thank you for your reply.
Yes it has a TCU.
I can't find "SOS call" to disable in these settings. Do you know where exactly is it? Also, how can I turn it off through DDT4ALL cluster settings?

Thank you very much.

**Reply by AndyIreland** [reply]
when you do a search with DDT which TCU ECU shows up? - here are the ECU's for Spring Ph1 - is yours a spring Ph1 or Ph2?  or is it a different renault you have?
![image](https://github.com/user-attachments/assets/bb369a2a-a0ad-4ae4-b25b-a886c1ddfebc)

**Reply by rxtarcher** [reply]
Good morning, 

I have a Renault Clio V Ph1, so I choose:

XJA - Clio V
TELEMATIC CONTROL UNIT/TCU
TCU_AIVC_V2.22
DTC ACTIVATION STATUS

When I am choosing "DTC ACTIVATION STATUS", the right column writes "NO DATA" everywhere. Even if try to change an option (like Aux-In Info/Tel-Mute-Output, or any other)  from "Activated" to "Deactivated" it doesn't work. It always goes back to "Activated". I have uploaded a detailed video to show you exactly what I mean.

https://youtu.be/Hm3U-LgYKeY

**Reply by rxtarcher** [reply]
Btw, I am using this ELM with WiFi and a laptop with windows 10 for ddt4all.
![1](https://github.com/user-attachments/assets/bb2bb81f-fd8a-48e2-9db3-5356ea544ace)
![2](https://github.com/user-attachments/assets/ae837064-1e99-4167-9c31-870c1e5db4e2)

**Reply by AndyIreland** [reply]
@rxtarcher - all makes sense now , - so you have a Clio5 - that is protected against changes by security gateway block on the OBD2 socket

**Reply by rxtarcher** [reply]
Okay, I see. So, is there something I can do?

**Reply by AndyIreland** [reply]
yes get a garage or official dealer to unlock the gateway with a Token or try bypass the gateway by wiring up directly to BCM module behind the dash , some people reckon it works .

Here is whole long thread on it , just skip to the bits you are interested in. good luck:
https://github.com/cedricp/ddt4all/discussions/956

**Comment by CarlabElettronica** [comment]
hi i'm trying to disable sos calling check light, i download new database and check, for all people want disable sgw can do with right cable and i test on megane 2020 and work good.

have nice day i report on monday when try again to remove check

**Reply by matzel82** [reply]
First… ist possible the Battery is empty or the ECU is broken or no Connection to Network

**Reply by CarlabElettronica** [reply]
yes, customer ask me to remove, on monday i will try again, any suggestion about wich ecu need to edit?

**Reply by AndyIreland** [reply]
Maybe its in DCM/TCU ECU - thats only a wild guess though to help you maybe go to the right area . I dont know for certain sorry:

![image](https://github.com/user-attachments/assets/66023a60-b2c0-4983-be83-1584615c6b29)

**Comment by Nicky-68** [comment]
Salve ho una sandero gpl 2021 e sarei interessato a disabilitare la centralina CHIAMATA SOS.
Non si accendono i led ROSSO e VERDE e c'è errore sul cruscotto "CHIAMATA SOS CONTROLLARE" e chiave inglese accesa.
Siete riusciti a disabilitare il modulo SOS mi dite come....?
Grazie in anticipo per l'aiuto.

**Reply by Tony2k** [reply]
Hi Nicky, in my case I disabled only the elements that were causing the error (see point 1 and 2). Your problem is probably caused by the buffer battery so I would start by trying to disable "Internal battery status". You can proceed by disabling everything and re-enabling one element at a time so as to also understand what is causing the problem.

**Comment by Nicky-68** [comment]
Grazie per la tua risposta Tony2K.
E molto probabile che sia un guasto.
Comunque spiegami come procedere......
- disabilito il punto 1 "Aux-In Info......."
- disabilito il punto 2 "Virgin configuraton"
- Premo il pulsante 3 e poi il pulsante 4 .............giusto..?
- Fine o devo fare altro....?
Grazie della tua disponibilità Proverò martedi e ti farò sapere in serata

**Comment by mohamedlrouabah1** [comment]
SOS warning disable -> 

![image](https://github.com/user-attachments/assets/6ad52010-a9d2-493d-8f35-282859ca2089)
